packages/medusa/src/api/routes/admin/auth/get-session.ts

```diff
@@ -53,7 +53,7 @@ import _ from "lodash"
 export default async (req, res) => {
   try {
     const userService: UserService = req.scope.resolve("userService")
-    const user = await userService.retrieve(req.user.userId)
+    const user = await userService.retrieve(req.user.userId || req.user.id)
```

Comment:
todo: Should use the nullish coalescing operation
```suggestion
    const user = await userService.retrieve(req.user?.userId ?? req.user?.id)
```

Reply:
Ok. Thanks for the feedback.